docs/pipelines/index.md

```diff
@@ -134,3 +134,209 @@ else
 <!-- endSnippet -->
 
 Use `ExecuteOutcomeAsync(...)` in high-performance scenarios where you wish to avoid re-throwing exceptions. Keep in mind that Polly's resilience strategies also make use of the `Outcome` struct to prevent unnecessary exception throwing.
+
+## Diagrams
+
+### Sequence diagram for a pipeline with retry and timeout
+
+<!-- snippet: resilience-pipeline-diagram-retry-timeout -->
+```cs
+// Let's create the following pipeline:
+// the inner strategy is a timeout,
+// the outer is a retry which is timeout-aware.
```

Comment:
I meant more actual prose in the document, rather than code comments.

It's also _much much_ easier to see the changes if you don't force push. Otherwise I have to just re-review everything as I can't diff.